src/compiler/diagnosticMessages.json

```diff
@@ -7604,5 +7604,9 @@
     "The value '{0}' cannot be used here.": {
         "category": "Error",
         "code": 18050
+    },
+    "Compiler option '{0}' cannot have an empty string.": {

```

Comment:
I don't find "have an empty string" to be grammatically correct; maybe "cannot be empty" instead or "cannot contain an empty string" instead?

Reply:
"cannot be empty" feels ambiguous (what does it mean for an option to be empty?  Is a whitespace-only string considered empty?  etc. etc.).  I would personally prefer "cannot be set to an empty string".

Reply:
Yeah, `Compiler option 'extends' cannot be empty` is confusing to me, especially in the case where the empty string appears in an extends array (i.e. `"extends": [""]`), because the array is not empty. I think `cannot contain an empty string` is more accurate, and understandable enough to cover both the empty string and empty string in array cases.

Reply:
@DanielRosenwasser thoughts on the error message?

Reply:
`Compiler option 'extends' cannot be given an empty string.` is good since it covers both the cases of `"extends": ""` and `"extends": [""]`